instaclone/src/components/SearchBar.js

```diff
@@ -5,29 +5,33 @@ const SearchBar = (props)=>{
     const style={
```

Comment:
We already know how I feel about creating a style object at the top of your component folder and then using them inline on jsx elements. :P

Which is to say, it does not look very clean. Please clean this up! :)